dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/programindicator/DefaultProgramIndicatorSubqueryBuilder.java

```diff
@@ -121,8 +122,9 @@ private String getAggregateClauseForPIandRelationshipType( ProgramIndicator prog
 
     private String getFrom( ProgramIndicator pi )
     {
-        return " FROM " + ANALYTICS_TABLE_NAME + "_" + pi.getAnalyticsType().getValue() + "_"
-            + pi.getProgram().getUid().toLowerCase() + " as " + SUBQUERY_TABLE_ALIAS;
+        String baseTableName = ANALYTICS_TABLE_NAME + "_" + pi.getAnalyticsType().getValue();
+
+        return " FROM " + PartitionUtils.getTableName( baseTableName, pi.getProgram() ) + " as " + SUBQUERY_TABLE_ALIAS;
```

Comment:
Tiny recommendation: change "FROM" to "from"

Reply:
Thanks @maikelarabori. I agree that I would like "from" better. However, "FROM" matches all the other uppercase SQL keywords in this class so I think we should change them all if we change any of them. To avoid further extending the scope of this PR, I suggest we change none of them for the moment.